docs/next/guides/building-and-testing/modules.md

```diff
@@ -615,112 +838,311 @@ import {
-registerCellType(HandsontableCellType);
-registerCellType(NumericCellType);
-registerCellType(PasswordCellType);
-registerCellType(TimeCellType);
-registerCellType(TextCellType);
+// use the locales
+new Handsontable(container, {
+  language: 'pl-PL',
+// rest of the settings
+});
+```
 
-registerPlugin(AutoColumnSize);
-registerPlugin(Autofill);
-registerPlugin(AutoRowSize);
-registerPlugin(BindRowsWithHeaders);
-registerPlugin(CollapsibleColumns);
-registerPlugin(ColumnSorting);
-registerPlugin(ColumnSummary);
-registerPlugin(Comments);
-registerPlugin(ContextMenu);
-registerPlugin(CopyPaste);
-registerPlugin(CustomBorders);
-registerPlugin(DragToScroll);
-registerPlugin(DropdownMenu);
-registerPlugin(ExportFile);
-registerPlugin(Filters);
-registerPlugin(Formulas);
-registerPlugin(HeaderTooltips);
-registerPlugin(HiddenColumns);
-registerPlugin(HiddenRows);
-registerPlugin(ManualColumnFreeze);
-registerPlugin(ManualColumnMove);
-registerPlugin(ManualColumnResize);
-registerPlugin(ManualRowMove);
-registerPlugin(ManualRowResize);
-registerPlugin(MergeCells);
-registerPlugin(MultiColumnSorting);
-registerPlugin(MultipleSelectionHandles);
-registerPlugin(NestedHeaders);
-registerPlugin(NestedRows);
-registerPlugin(ObserveChanges);
-registerPlugin(PersistentState);
-registerPlugin(Search);
-registerPlugin(TouchScroll);
-registerPlugin(TrimRows);
-registerPlugin(UndoRedo);
+And that is all! You can use the PL-pl locale!
 
-new Handsontable(container, {});
+### Optimizing moment.js locales
+
+If you want to decrease the bundle size even more you can also focus on optimizing moment.js locales. There are different methods of doing so but this guide focuses on using the webpack's [IgnorePlugin](https://webpack.js.org/plugins/ignore-plugin/). For another option, you can check [this tutorial](https://github.com/jmblog/how-to-optimize-momentjs-with-webpack) directly.
+
+Moment.js can be heavyweight because when you type `var moment = require('moment')` in the code you get all locales as default. To be more selective with your choice you can first, use the `IgnorePlugin` that removes all locales:
+
+```js
+const webpack = require('webpack');
+module.exports = {
+  //...
+  plugins: [
+    // Ignore all locale files of moment.js
+    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+  ],
+};
 ```
+
+And then explicitly load selected locales:
+
+```js
+import moment from 'moment';
+import 'moment/locale/ja';
+import Handsontable from 'handsontable/base';
+import { registerCellType, DateCellType } from 'handsontable/cellTypes';
+
+moment.locale('ja');
+registerCellType(DateCellType);
+
+new Handsontable(container, {
+  type: 'date',
+});
+```
+
+## Using modules with wrappers
+
+You can use modules with Handsontable's [React](@/guides/integrate-with-react/react-installation.md), [Angular](@/guides/integrate-with-angular/angular-installation.md), and [Vue 2](@/guides/integrate-with-vue/vue-installation.md) wrappers.
+
+### Using modules with React
```

Comment:
I don't know but maybe we could link or move that part of the documentation about modules to the framework part? 🤔

Reply:
Yeah, I was thinking about that. I think you're right: framework users probably look mainly at the "Integrate with React/Vue/Angular" section (ofc we don't have any analytics data to support that assumption, though), so info about modules should definitely be there. And, we can link to those sections from the "Modules" topic.

Separate topics also nicely display in search:
<img width="342" alt="1" src="https://user-images.githubusercontent.com/29247866/140053700-2a6bc97a-7666-4310-8572-14d885a2f860.png">

Here's the change: a5133d1